inc/console/abstractcommand.class.php

```diff
@@ -90,7 +89,7 @@ protected function initDbConnection() {
       global $DB;
 
       if ($this->requires_db && (!($DB instanceof DB) || !$DB->connected)) {
-         throw new RuntimeException(__('Unable to connect to database.'));
+         throw new \Symfony\Component\Console\Exception\RuntimeException(__('Unable to connect to database.'));
```

Comment:
This one sounds strange :D

Reply:
This is why this rule exists. Many libs uses same names for exceptions (like `RuntimeException` or `NotFoundException`). Forcing usage of full path will prevent ambiguity/mistakes.